lib/galaxy/tool_util/linters/tests.py

```diff
@@ -9,8 +9,7 @@
 from ..verify import asserts
 
 
-# Misspelled so as not be picked up by nosetests.
-def lint_tsts(tool_xml, lint_ctx):
+def lint_tests(tool_xml, lint_ctx):
```

Comment:
Could this be in use with people using tool-util as a library ?

Reply:
Not by planemo, ephemeris or cwltool. Any other I should check?